node/pkg/vaa/payloads_test.go

```diff
@@ -90,7 +90,7 @@ func TestBodyTokenBridgeRegisterChainSerialize(t *testing.T) {
 	module := "test"
 	addr := Address{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4}
 	bodyTokenBridgeRegisterChain := BodyTokenBridgeRegisterChain{Module: module, ChainID: 1, EmitterAddress: addr}
-	expected := "000000000000000000000000000000000000000000000000000000007465737401000000010000000000000000000000000000000000000000000000000000000000000004"
+	expected := "00000000000000000000000000000000000000000000000000000000746573740100010000000000000000000000000000000000000000000000000000000000000004"
```

Comment:
The only thing we change is the removal of target chain id?

Reply:
Yes, we moved the `targetChainId` fields from the payload to the VAA header, I forgot to fix the `RegisterChain` payload before.